content/build/cache/backends/s3.md

```diff
@@ -13,8 +13,8 @@ aliases:
 { .restricted }
 
 The `s3` cache storage uploads your resulting build cache to
-[Amazon S3 file storage service](https://aws.amazon.com/s3/),
-into a specified bucket.
+[Amazon S3 file storage service](https://aws.amazon.com/s3/)
+or other S3-compatible services.
```

Comment:
May be worth directly calling out [minio](https://min.io/) here? It's [open-source](https://github.com/minio/minio), and also, it's what we use in the buildkit CI to actually test the S3 cache :smile:

Reply:
ah that's a great suggestion

Reply:
added ✅